src/Geometry/DiscreteModels.jl

```diff
@@ -415,12 +415,11 @@ function DiscreteModelFromFile(filename::AbstractString)
 end
 
 function DiscreteModelFromFile(filename::AbstractString,::Any)
-  @notimplemented
+  error("File type unknown (did you forget to load GridapGmsh?)")
```

Comment:
Can you check that the extension in `filename` is indeed `.msh` to include the hint about loading GridapGmsh ?

Reply:
I adapted the warning to be more generic